Makefile

```diff
@@ -51,7 +51,7 @@ test_failed:
 	$(DOCKER_PREFIX) py.test --lf $(ARGS)
 
 initialize_db:
-	$(DOCKER_PREFIX) python manage.py reset_db
+	$(DOCKER_PREFIX) python manage.py reset_db --noinput
```

Comment:
If you're not using this target for the `initialize_docker` one, then I'd rather not have the `--noinput` ;)